doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,104 @@
+.. rn:: 8.0.16-7
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+|Percona Server| |release| is now the current GA release in the 8.0
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0
+Community Edition
```

Comment:
8.0.16, this release doesn't have the features from 8.0.17 that's already out

Reply:
Specified 8.0.16

Reply:
includes all the features and bug fixes

Reply:
added "and bug fixes" to the description.